docs/api/sdk/README.md

```diff
@@ -402,33 +402,44 @@ Use the `exception` type for errors and crashes.
+| `type`                     | string  | No       | Type of the exception          |
+| `message`                  | string  | No       | Error message text             |
+| `frames`                   | array   | Yes      | Array of stackframe objects    |
+| `signal`                   | string  | Yes      | BSD termination signal         |
+| `thread-name`              | string  | Yes      | Name of thread                 |
+| `thread-sequence`          | Int     | Yes      | Thread number                  |
+| `operatingSystemBuild`     | string  | Yes      | Operating system build number  |
+| `cpuArch`                  | string  | Yes      | The CPU architecture           |
 
 `thread` objects
 
 Each thread object contains further fields.
 
-| Field    | Type   | Optional | Comment                     |
-| -------- | ------ | -------- | --------------------------- |
-| `name`   | string | Yes      | Name of thread              |
-| `frames` | array  | Yes      | Array of stackframe objects |
+| Field      | Type   | Optional | Comment                     |
+| ---------- | ------ | -------- | --------------------------- |
+| `name`     | string | Yes      | Name of thread              |
+| `frames`   | array  | Yes      | Array of stackframe objects |
+| `sequence` | Int    | Yes      | Thread number               |
 
 `frame` objects
 
 Each frame object contains further fields.
 
-| Field         | Type   | Optional | Comment                        |
-| ------------- | ------ | -------- | ------------------------------ |
-| `line_num`    | number | Yes      | Line number of the method      |
-| `col_num`     | number | Yes      | Column number of the method    |
-| `module_name` | string | Yes      | Name of the originating module |
-| `file_name`   | string | Yes      | Name of the originating file   |
-| `class_name`  | string | Yes      | Name of the originating class  |
-| `method_name` | string | Yes      | Name of the originating method |
+| Field               | Type   | Optional | Comment                        |
+| ------------------- | ------ | -------- | ------------------------------ |
+| `line_num`          | number | Yes      | Line number of the method      |
+| `col_num`           | number | Yes      | Column number of the method    |
+| `module_name`       | string | Yes      | Name of the originating module |
+| `file_name`         | string | Yes      | Name of the originating file   |
+| `class_name`        | string | Yes      | Name of the originating class  |
+| `method_name`       | string | Yes      | Name of the originating method |
+| `binary_name`       | string | Yes      | Name of the iOS binary image   |
+| `symbol_address`    | string | Yes      | Address to symbolicate         |
+| `offset`            | string | Yes      | Byte offset                    |
+| `binary_address`    | string | Yes      | Binary load address            |
+| `frame_index`       | Int    | Yes      | The stack frame number         |
```

Comment:
Do we need frame index explicitly? Can this be inferred from the sequence of the `frames` array? This is how it's done for Android too. The SDK makes sure the frames are in order of what needs to be shown on UI